server/routes/courtRegister/addNewCourtSummaryValidator.ts

```diff
@@ -25,15 +25,15 @@ function asAddCourt(form: AddNewCourtForm): AddCourt {
   const contacts: InsertCourtBuildingContact[] = [
     {
       type: 'TEL',
-      detail: form.telephonenumber,
+      detail: form.telephonenumber as string,
     },
```

Comment:
these can never be null by the time it gets to here - we validate that they can't be null. So a few choices:
1) Check at runtime anyway and throw Error
2) Just cast to `string` as above (or `!`)

Doing 1) might create a lot of unnecessary code noise